README.md

```diff
@@ -1,21 +1,36 @@
 # Nunjucks Isomorphic loader for webpack
 
-Nunjucks loader for webpack, supporting both javascript templating and generating
-static HTML files through the HtmlWebpackPlugin.
+Nunjucks loader for webpack, supporting both javascript templating and generating static HTML files through the HtmlWebpackPlugin.
 
 Completely rewritten, now (mostly) error free. Supports context file (.js only) for passing data.
 
-This loader only supports `.njk` extension!
+This loader supports the following extensions: `.njk`, `.nunjucks`, `.html`, `.tpl`, `.tmpl`
 
-**Usage**
+## Installation
 
-In webpack.config.js
+This loader needs the [`nunjucks`](https://www.npmjs.com/package/nunjucks) and [`nunjucks-loader`](https://www.npmjs.com/package/nunjucks-loader) as peer dependencies.

```

Comment:
As far as I'm aware, the `nunjucks-loader` should be installed automatically when you `npm install nunjucks-isomorphic-loader`, due to it's placement in the dependencies section of the `package.json`. In NPM 1 & 2, the peerDependencies will also be downloaded automatically, however this behaviour is changing in NPM 3.

More details can be found here: https://docs.npmjs.com/files/package.json

Reply:
Yeah, I had the following warnings with yarn:

```
yarn add nunjucks-isomorphic-loader -D
yarn add v1.3.2
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "nunjucks-isomorphic-loader > nunjucks-loader@2.4.8" has unmet peer dependency "nunjucks@>= 2.5.0 <= 4.0.0".
warning " > nunjucks-isomorphic-loader@2.0.2" has unmet peer dependency "nunjucks@>= 2.5.0 <= 4.0.0".
```

So I had to manually install those two dependencies by myself like I indicated in the docs

Reply:
Ah-ha! Yes, that makes more sense. Okay, I'll merge this in, and tweak it a little.

I think it's best that we make it clear that nunjucks is a peer dependency that only needs to be installed if you do not have it already, and supports nunjucks versions 2.5 - 4.0